acme/setup.py

```diff
@@ -22,15 +22,14 @@
 ]
 
 # env markers in extras_require cause problems with older pip: #517
+# Keep in sync with conditional_requirements.py.
 if sys.version_info < (2, 7):
     install_requires.extend([
         # only some distros recognize stdlib argparse as already satisfying
         'argparse',
-        'mock<1.1.0',
```

Comment:
After much further discussion on IRC and at EFF, our revised conclusion is: we'd like to move the mock dependency out of the main setup.py, but we aren't going to rush shipping that way, so 0.2.0 is going out without this change and we can do it in a more relaxed manner for 0.2.1 on Monday.